subprojects/docs/src/docs/userguide/publishingIvy.adoc

```diff
@@ -31,23 +31,23 @@ A published Ivy module can be consumed by Gradle (see <<declaring_dependencies>>
 
 
 [[publishing_ivy:plugin]]
-=== The “`ivy-publish`” Plugin
+=== The Ivy Publish Plugin
 
-The ability to publish in the Ivy format is provided by the “`ivy-publish`” plugin.
+The ability to publish in the Ivy format is provided by the Ivy Publish Plugin.
 
-The “`publishing`” plugin creates an extension on the project named “`publishing`” of type api:org.gradle.api.publish.PublishingExtension[]. This extension provides a container of named publications and a container of named repositories. The “`ivy-publish`” plugin works with api:org.gradle.api.publish.ivy.IvyPublication[] publications and api:org.gradle.api.artifacts.repositories.IvyArtifactRepository[] repositories.
+The `publishing` plugin creates an extension on the project named `publishing` of type api:org.gradle.api.publish.PublishingExtension[]. This extension provides a container of named publications and a container of named repositories. The Ivy Publish Plugin works with api:org.gradle.api.publish.ivy.IvyPublication[] publications and api:org.gradle.api.artifacts.repositories.IvyArtifactRepository[] repositories.
```

Comment:
I guess "`publishing` plugin" should be "Ivy Publish Plugin"? Or you could probably just refer to "The plugin ..."